app/src/interfaces/list-m2a/index.ts

```diff
@@ -52,6 +52,17 @@ export default defineInterface({
 				default_value: 15,
 			},
 		},
+		{
+			field: 'allowDuplicates',
+			name: '$t:allow_duplicates',
```

Comment:
Let's make sure to add this translation in `app/src/lang/translations/en-US.yaml` 👍

Reply:
Added 👍